templates/base/navbar.html

```diff
@@ -1,16 +1,11 @@
 
 
      <nav class="navbar navbar-expand-lg navbar-dark  bg-dark">
-        <a class="navbar-brand" href="#">Ketabdan</a>
+        <a class="navbar-brand" href="../../">Ketabdan</a>
```

Comment:
This is still not quite right. Check [here](https://stackoverflow.com/questions/19742116/link-to-home-page-from-another-page-with-sub-page-anchor-i-e-somebookmark)

Also you forgot to push your screenshot